guacamole-existing-vnet/azuredeploy.json

```diff
@@ -0,0 +1,236 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
```

Comment:
As per my comment above (https://github.com/Azure/azure-quickstart-templates/pull/1905#issuecomment-216701584) I'm not planning to, as it's a lot of work for something that's more like an edge case IMHO. But I'll gladly accept input from you and the community.